### ISSUE TITLE
Elemental Seal accuracy issues with certain monsters

### ISSUE THREAD

**Issue by Steam-arch** [EdenServer/community#4921]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [ ] I have provided reproducible steps. (No "as the title says" posts please. Provide reproducible steps even if it seems like a waste of time.)
- [ ] I have provided my client version in the details. (type /ver into your game window)
- [ ] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details

Once again im sorry for a vague A5 report, and "lack of proof" but id like to open a discussion about Elemental Seal Sleep accuracy when used on (atleast) monsters :

Snow Devil (ENM When hell Freezes over)
Aw'zdei (When angels fall CoP fight)
Eldertaur/Mindertaur (Brothers ENM)

The case is that Elemental Seal does not seem to increase the accuracy of the Sleep1/2/ga like its supposed to. Its basically rolling a dice when sleeping these monsters with elemental seal and its lands maybe 50-80% (?) of the time. I do When Hell freezes Over on rdm/blm occasionally and ES sleepga is always like described below "super sketch" and you usually have a resist or two and you have to pop chainspell to clear the wave, get a 2nd wave of 3 and you might aswell warp out. Sleeping the Aw´zdei with ES sleepga also has a similar behaviour and is  " sketch", normal pots are coded to resist sleep aswell and resist close to 50% of the time with enfeebling +310 so all 3 monsters propably have that in common. 

This issue does make the ENMs much harder and more rng than they are supposed to be. I do believe ES sleep should be capped landing rate on these mobs, but cant really provide proof to this claim.

![image](https://user-images.githubusercontent.com/69089249/155128057-03a928dd-f5c6-4e20-b679-3054dcece2fb.png)

Two gentlemen were discussing Brothers ENM on discord about ES sleep issue with the taurs.

![image](https://user-images.githubusercontent.com/69089249/155125753-60ef1c64-3e2d-4322-894f-1b30d37f6f54.png)
Mozzy suggested this, which could lower the rate of which ES Sleep lands. Which might be right because all 3 monsters are coded to somewhat resist sleep (+taurus is dark based, but ES +250 macc should still propably overcome these with normal gear and cap the landing rate).

Basically im asking are the unusual ES Sleep resists on these monsters actually intended since something seems off. I can make Elegy land no problem on khimaira with ES, but I cant consistently land an ES sleepga on a few enm snoll 😵

**Comment by Raulfin** [comment]
on the When angels fall fight as a rdm/blm i was able to ES sleepga nearly every time in prolly 10 fights, may have had 1-2 resists from about 20 casts, but can't remember anything specific.
on 1 of the fights we had a 2nd rdm that had full Enfeeb merits that was able to land sleepga without using ES.

**Comment by 59blargedy** [comment]
this should be mainly resolved by the trait and monster resistance changes in the November 21, 2022 update.
